db/import.go

```diff
@@ -107,14 +107,11 @@ func (db *Database) importDoc(docid string, body Body, isDelete bool, existingDo
 		body = Body{}
 	}
 
-	newDoc := &Document{
-		ID:      docid,
-		Deleted: isDelete,
-	}
-
+	var newDoc *IncomingDocument
```

Comment:
If we don't need this scoped (now we're using docid everywhere) - let's create it when we call `newDoc, err := body.ToIncomingDocument()`